src/scripts/main.js

```diff
@@ -1,3 +1,306 @@
 'use strict';
 
-// write code here
+const table = document.body.querySelector('table');
+const thead = document.body.querySelector('thead');
+const tbody = document.body.querySelector('tbody');
+
+let clickCount = 0;
+
+thead.addEventListener('click', (e) => {
+  const target = e.target.closest('th');
+  const cellIndex = target.cellIndex;
+  const trBody = tbody.querySelectorAll('tr');
+
+  if (!target) {
+    return;
+  }
+
+  clickCount++;
+
+  const sortedTr = [...trBody].sort((a, b) => {
+    let value1 = a.children[cellIndex].textContent;
+    let value2 = b.children[cellIndex].textContent;
+
+    if (clickCount % 2 === 0) {
+      value1 = b.children[cellIndex].textContent;
+      value2 = a.children[cellIndex].textContent;
+    }
+
+    if (!isNaN(+value1)) {
+      return value1 - value2;
+    } else if (value1[0] === '$') {
+      return value1.slice(1).split(',').join('') - value2
```

Comment:
you are doing subtraction of strings, also better to move this transformation to a separate function